libusb/libusb.h

```diff
@@ -335,6 +335,8 @@ enum libusb_descriptor_type {
 #define LIBUSB_DT_SS_ENDPOINT_COMPANION_SIZE	6
 #define LIBUSB_DT_BOS_SIZE			5
 #define LIBUSB_DT_DEVICE_CAPABILITY_SIZE	3
+#define LIBUSB_DT_INTERFACE_ASSOCIATION_SIZE			8
+#define LIBUSB_SMALLEST_DESCRIPTOR_SIZE			2	/* length & desc type, like usbi_descriptor_header */
```

Comment:
Do we want this one in the public header?

Reply:
I have no strong feeling. I'll put them wherever you want.